samples/sqltransport/native-integration/Version_3/Receiver/PlaceOrderHandler.cs

```diff
@@ -4,7 +4,7 @@
 
 public class PlaceOrderHandler : IHandleMessages<PlaceOrder>
 {
-    static ILog log = LogManager.GetLogger<LegacyOrderDetectedHandler>();
+    static ILog log = LogManager.GetLogger<PlaceOrderHandler>();
```

Comment:
After running sample first I thought that this is a bug in v3 and wrong handler is called, turned out to be something more trivial